### PR TITLE
added icon for breezy weather

### PR DESCRIPTION
**Pull request by ankurs**
currently weather shows '?' for "breezy" weather

### CODE REVIEW

File: segments/weather.sh [hunk 1]
```diff
@@ -160,6 +160,10 @@ __get_condition_symbol() {
 			#echo "﹌"
 			echo "〰"
 			;;
+        "breezy")
```
Comment (erikw) [inline segments/weather.sh]: The indentation is off, compare with the rest of the file :).  Make sure to use hard tabs for indentation.

Comment (ankurs) [inline segments/weather.sh]: ohh, i have expand tab set in vim, this looked fine in my local copy, will update this request with tabs

**Comment by ankurs** [comment]
updated!

**Comment by erikw** [comment]
Thanks :)

**Comment by ankurs** [comment]
you are welcome!

**Pull request merged**